client/src/components/public/calendar/ConfirmEventType.vue

```diff
@@ -160,14 +159,88 @@ export default {
         ...mapGetters('publicEvent', {
             eventType: getters.GET_EVENT_TYPE,
             publicEvent: getters.GET_PUBLIC_EVENT
-        })
+        }),
+        startDateFormatted() {
+            return moment(this.publicEvent.startDate).format(
+                'dddd, YYYY-MM-DD, HH:mm'
```

Comment:
HH:mm:ss

Reply:
I think this date shouldn't contain seconds, they will not be used and having only hours and minutes it would be easier to work with this date. And even if we would need seconds, we can get them in the future as momentjs let us format date having written ".format(... :ss)" in any place on the code